internal/pkg/logs/service.go

```diff
@@ -0,0 +1,85 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package logs
+
+import (
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/aws/copilot-cli/internal/pkg/aws/cloudwatchlogs"
+	"github.com/aws/copilot-cli/internal/pkg/aws/sessions"
+	"github.com/aws/copilot-cli/internal/pkg/config"
+	"github.com/aws/copilot-cli/internal/pkg/term/log"
+)
+
+const (
+	logGroupNamePattern = "/copilot/%s-%s-%s"
+)
+
+type logGetter interface {
+	TaskLogEvents(logGroupName string,
+		streamLastEventTime map[string]int64,
+		opts ...cloudwatchlogs.GetLogEventsOpts) (*cloudwatchlogs.LogEventsOutput, error)
+}
+
+// ServiceLogs wraps service that writes service log events to a writer.
+type ServiceLogs struct {
```

Comment:
What do you think of `ServiceClient` instead? so that it reads `ecslogging.ServiceClient`?

```go
// ServiceClient retrieves the logs of an Amazon ECS service.
type ServiceClient struct {
```